src/app/shared/utils/utils.ts

```diff
@@ -129,4 +130,57 @@ export class Util {
   });
   return updatedUsers;
 }
+
+  /**
+   * This method returns union message for the workshop updating
+   * @param payload Object
+   * @returns string
+   */
+    public static getWorkshopMessage(payload) {
+      let finalMessage = {text: '', type: 'success'};
+      let messageArr = [];
+
+      let isInvalidCoverImage = false;
+      let isInvalidGaleryImages = false;
+      let statuses, invalidImages;
+      
+      if (payload.uploadingCoverImageResult) {
+        isInvalidCoverImage = !payload.uploadingCoverImageResult.result.succeeded
+      }
+
+      if (payload.uploadingImagesResults) {
+        statuses = Object.entries(payload.uploadingImagesResults?.results);
+        invalidImages = statuses.filter((result) => !result[1]['succeeded']);
+        isInvalidGaleryImages = !!invalidImages.length;
+      }
+
+      messageArr.push(`Гурток оновлено!`);
+
+      if (isInvalidCoverImage) {
+        const coverImageErrorMsg = payload.uploadingCoverImageResult?.result.errors
+        .map((error) => `"${CodeMessageErrors[error.code]}"`)
+        .join(', ');
```

Comment:
pelase please please:) can you fix the alignment of the methods? add one space before dot :)))